### ISSUE TITLE
Piston duplication does not work

### ISSUE THREAD

**Issue by KujouMolean** [MultiPaper/MultiPaper#275]
This is definitely caused by MultiPiper, or more specifically the 0181 patch. Compiling with 0181 patch, the duplication breaks. Remove it, duplication works.

![Snipaste_2022-10-10_08-08-11](https://user-images.githubusercontent.com/54140989/194785713-a95dae58-f5e2-47a9-9307-4377c0c21fbc.png)

**Comment by xymb-endcrystalme** [comment]
https://github.com/xymb-endcrystalme/MultiPaperPrivateDontShare/blob/master/patches/server/0272-Make-TNT-dupers-work-again.patch

That's basically what I did. Removed MP code, put Paper code back in.

**Comment by MrIvanPlays** [comment]
> [xymb-endcrystalme/MultiPaperPrivateDontShare@`master`/patches/server/0272-Make-TNT-dupers-work-again.patch](https://github.com/xymb-endcrystalme/MultiPaperPrivateDontShare/blob/master/patches/server/0272-Make-TNT-dupers-work-again.patch?rgh-link-date=2023-01-12T12%3A37%3A26Z)
> 
> That's basically what I did. Removed MP code, put Paper code back in.

That's not the right solution. Stuff should be in sync...

**Comment by xymb-endcrystalme** [comment]
I agree that it's not the right solution. The servers aren't crashing so I can't complain.

On a bigger world TNT dupers rarely tick on two servers, so they don't break often.
It works for me, but it's definitely not a recommended solution, just a kludge.
